imkar/scattering/analytical.py

```diff
@@ -0,0 +1,186 @@
+import numpy as np
+import pyfar as pf
+
+import numbers
+
+
+def rectangular(frequencies, incident_angles, rectangle_width, gap_width,
+                height, speed_of_sound=346.18):
+    """
+    This function computes the scattering coefficient of periodic rectangular
+    profiles, assuming that the extent is infinite.
+
+    This function is based on Ducourneau's method described by Embrechts
+    [#]_.
+
+    Parameters
+    ----------
+    frequencies : array, float
+        Vector of frequencies the scattering coefficient is computed at.
+    incident_angles : array, float
+        Vector or value of incident angles in degree for the scattering
+        coefficient calculation. Angles between 0°-90° are allowed.
+    rectangle_width : float
+        Width of the rectangulars in m.
+    gap_width : float
+        Width of the gap in m.
```

Comment:
I am not sure, if this is the best interface. Maybe something more akin to pulse width might be an easier to understand interfac (i.e. wave length of one spatial section and a percentage of how much is raised).

Reply:
How about using `total_width`, `gap_width`, and `gap_height` as parameter names?